lib/manager/travis/package.ts

```diff
@@ -93,7 +17,7 @@ export async function getPackageUpdates(
   if (!supportPolicy?.length) {
     return [];
   }
-  checkPolicies();
+  policies = policies || getPolicies();
```

Comment:
You've removed the policy update check. This can cause issues with long running renovate app / on-premise (sometimes multiple weeks).

Please add a refresh check as before.

Reply:
What about just computing policies for each call? I don't think memoization improves performance sufficiently here.